lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -68,7 +68,7 @@ describe('PerfCategoryRenderer', () => {
   it('renders the sections', () => {
     const categoryDOM = renderer.render(category, sampleResults.categoryGroups);
     const sections = categoryDOM.querySelectorAll('.lh-category > .lh-audit-group');
-    assert.equal(sections.length, 4);
+    assert.equal(sections.length, 5);
```

Comment:
should also add some tests to ensure rendering is happening, etc. We already have tests for the table details renderer, so it can probably stay relatively light. Starting suggestion:
```js
describe('budgets', () => {
  it('renders a performance budget', () => {
    const categoryDOM = renderer.render(category, sampleResults.categoryGroups);

    const budgetsGroup = categoryDOM.querySelector('.lh-audit-group.lh-audit-group--budgets');
    assert.ok(budgetsGroup);
    const budgetTable = budgetsGroup.querySelector('#performance-budget.lh-table');
    assert.ok(budgetTable);

    const lhrBudgetEntries = sampleResults.audits['performance-budget'].details.items;
    const tableRows = budgetTable.querySelectorAll('tbody > tr');
    assert.strictEqual(tableRows.length, lhrBudgetEntries.length);
  });

  it('does not render a budget table when performance-budget audit is notApplicable', () => {
    const budgetlessCategory = JSON.parse(JSON.stringify(category));
    const budgetRef = budgetlessCategory.auditRefs.find(a => a.id === 'performance-budget');
    budgetRef.result.scoreDisplayMode = 'notApplicable';
    delete budgetRef.result.details;

    const categoryDOM = renderer.render(budgetlessCategory, sampleResults.categoryGroups);
    const budgetsGroup = categoryDOM.querySelector('.lh-audit-group.lh-audit-group--budgets');
    assert.strictEqual(budgetsGroup, null);
  });
});
```